rest_framework_jwt/utils.py

```diff
@@ -115,21 +127,30 @@ def jwt_decode_handler(token):
     )
 
 
-def jwt_response_payload_handler(token, user=None, request=None):
+def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
     """
     Returns the response data for both the login and refresh views.
     Override to return a custom response such as including the
     serialized representation of the User.
 
     Example:
 
-    def jwt_response_payload_handler(token, user=None, request=None):
+    def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
         return {
             'token': token,
             'user': UserSerializer(user, context={'request': request}).data
         }
 
     """
-    return {
+    data = {
```

Comment:
It's just the response handler (not serializer, the serializer provides more data, if someone using our library would like to use more features - see the example above). By default in django-rest-framework-jwt the response_payload_handler only returns `token` - and now it returns `permanent_token` and `device_id` as well when user signing in.